docs/source/notebooks/tutorial.ipynb

```diff
@@ -4,40 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tutorial"
+    "# Tutorial\n",
+    "\n",
+    "Computer simulations of the Earth’s climate and weather generate huge amounts of data. These data are often persisted on HPC systems or in the cloud across multiple data assets of a variety of formats (netCDF, zarr, etc…). Finding, investigating, loading these data assets into compute-ready data containers costs time and effort. The data user needs to know what data sets are available, the attributes describing each data set, before loading a specific data set and analyzing it. Intake ESM is a Python package that aims to provide a higher-level interface to searching and loading Earth System Model (ESM) data archives, such as CMIP6. "
```

Comment:
"Computer simulations of the Earth’s climate and weather generate huge amounts of data. These data are often persisted on HPC systems or in the cloud across multiple data assets in a variety of formats (netCDF, zarr, etc…). Finding, investigating, and loading these data assets costs time and effort. The end user needs to not only know what data sets are available, but also the attributes describing each data set, before loading a specific data set and analyzing it.  The intake-esm package aims to provide a simple, high-level interface to searching and loading Earth System Model (ESM) data archives, such as CMIP6. "

Reply:
Just small suggestions, all subjective.